docs/FURPS/jump-to-screen-shell.md

```diff
@@ -0,0 +1,28 @@
+# Jump to screen (Shell) FURPS ([#17971](https://github.com/status-im/status-desktop/issues/17971))
+
+## Functionality
+- Support navigation to communities, chats, accounts, or dApps through a unified launcher interface.
+- Display recent items with live metadata (e.g., unread count, media preview, timestamps).
+- Order from most recent. When item is clicked, it moves to the top.
```

Comment:
I already expressed my mixed feelings on the PR related to this behavior: `It's a bit annoying to me that the grid order changes each time I click a new item. It's nice to have the most recents on top but then it makes it difficult to find a specific one since it's never at the same position. Maybe we could have the possibility of ordering by recents or not, so user can decide`

Reply:
Good point. I don't have hard feelings either way. 

I hope more people can chime in and express their opinion so we can decide if we drop that functionality

Reply:
Yeah... this was requested by Ben; I'm also not very keen on the items jumping around; I'd much rather see them ordered either by "notifications" first, and then alphabet